recipes/icu/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+./configure --prefix="$PREFIX" \
+    --disable-samples \
+    --disable-tests \
```

Comment:
Nevermind, I played with this locally. There don't appear to be lots of tests, but they are run. This actually has nothing to do with tests, but test data.